utils/controleval.py

```diff
@@ -265,17 +353,21 @@ def parse_arguments():
     stats_parser.add_argument(
         '-s', '--status', default='all',
         help="status used to filter the controls list output")
+    prometheus_parser = subparsers.add_parser(
+        'prometheus',
+        help="calculate and return benchmarks metrics in Prometheus format")
+    prometheus_parser.add_argument(
+        '-p', '--products', nargs='+', required=True,
+        help="list of products to process the respective controls files")
+    prometheus_parser.add_argument(
```

Comment:
I think this should be required as the command below returns nothing.

```
./utils/controleval.py prometheus -p rhel8 rhel9 rhel7
```

Reply:
Or without `-f` it should output to stdout

Reply:
The last commit improves the default behavior. Please, see more details in the commit message.